samples/sensor/amg88xx/sample.yaml

```diff
@@ -1,12 +1,24 @@
+common:
+  tags: sensors
 sample:
   description: AMG88XX sensor sample
   name: AMG88XX sample
 tests:
-  sample.sensor.amg88xx:
+  sample.sensor.amg88xx.amg88xx_grid_eye_eval_shield:
+    build_only: true
+    depends_on: arduino_i2c arduino_gpio
+    platform_allow: pan1780_evb pan1770_evb pan1781_evb pan1782_evb nrf52840dk_nrf52840 nrf52833dk_nrf52833 nrf52dk_nrf52832 reel_board_v2 reel_board ubx_evkninab3_nrf52840 ubx_bmd340eval_nrf52840 ubx_bmd300eval_nrf52832 ubx_evkannab1_nrf52832 ubx_bmd360eval_nrf52811 ubx_evkninab4_nrf52833 ubx_bmd330eval_nrf52810
```

Comment:
What are the expectations of this list in platform_allow? This driver is platform independent, board specific dependencies are line above.

Reply:
I did this, because I found no other way, to set the pull-up's from the shield overlay on the board underneath, like discussed here: https://github.com/zephyrproject-rtos/zephyr/issues/53175 I get a lot of errors in the twister run, if ``i2c0_default`` is not available. So I simply added the boards from the list, that have ``i2c0_default`` available. Is it also possible to add ``i2c0_default`` here on the ``depends_on`` line and remove the ``paltform_allow`` list completely?